Vester/Tests/Network/Remove-vmnetwork.Vester.ps1

```diff
@@ -0,0 +1,36 @@
+# Test file for the Vester module - https://github.com/WahlNetwork/Vester
+# Called via Invoke-Pester VesterTemplate.Tests.ps1
+
+# Test title, e.g. 'DNS Servers'
+$Title = 'Remove default vmnetwork'
+
+# Test description: How New-VesterConfig explains this value to the user
+$Description = 'VM Network is the default portgroup and that has to be removed'

```

Comment:
Again, this seems more relevant to opentable policy rather than something that MUST be done in vmware. We should modify the wording of this file to "allow for" this type of config, or else perhaps even move this particular code to ot-vester as opentable specific.